modules/tests/test/dataobject/operations.go

```diff
@@ -32,9 +43,29 @@ func WaitForSuccessfulDataSource(cdiClientSet cdicliv1beta1.CdiV1beta1Interface,
 	})
 }
 
-func IsDataVolumeImportSuccessful(cdiClientSet cdicliv1beta1.CdiV1beta1Interface, namespace string, name string) bool {
-	dataVolume, err := cdiClientSet.DataVolumes(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+func WaitForBoundPVC(k8sClient *kubernetes.Clientset, namespace, name string, timeout time.Duration) error {
```

Comment:
What's the difference between a successful DV and a bound PVC? `WaitForSuccessfuleDataVolume` does not check if the PVC is bound.

Reply:
the function is no longer needed and was deleted